src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,46 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
```

Comment:
you can use destructurization here like this:
```suggestion
export const MovieCard = ({ movie }) => {
  const {
    prop1,
    prop2,
    prop3,
    etc,
  } = movie
  return (
```
and of course if you use such a method, you should close {} in the end of the code